### ISSUE TITLE
URGENT: Issue with jitpack

### ISSUE THREAD

**Issue by miarevalo10** [mieszko4/react-native-zoom-us#324]
We're seeing the same error reported here https://github.com/mieszko4/react-native-zoom-us/issues/311 and explained in https://github.com/mieszko4/react-native-zoom-us/issues/311#issuecomment-1789812538

```
 > Could not find mobilertc-5.13.10.12579.aar (com.github.zoom-us-community.jitpack-zoom-us:mobilertc:5.13.10.12579).
        Searched in the following locations:
            https://www.jitpack.io/com/github/zoom-us-community/jitpack-zoom-us/mobilertc/5.13.10.12579/mobilertc-5.13.10.12579.aar
```

Last time this was fixed by just releasing a new version https://github.com/mieszko4/react-native-zoom-us/issues/311#issuecomment-1791851275

**Comment by mieszko4** [comment]
Argh.. I will redeploy tomorrow

**Comment by lamlengend98** [comment]
Now, this still happen, right?

**Comment by mieszko4** [comment]
Should be fixed in `6.18.2`
